bin/build-locales

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env node
+/* eslint-disable no-var, no-console */
+// We can't use strict mode because po2json causes
+// referenceErrors.
+
+
+const fs = require('fs');
+const os = require('os');
+const path = require('path');
+const config = require('config');
+const po2json = require('po2json');
+const glob = require('glob');
+const shelljs = require('shelljs');
+const chalk = require('chalk');
+
+const appName = config.get('appName');
+
+if (!appName) {
+  console.log(
+    chalk.red('Please specify the appName with NODE_APP_INSTANCE'));
+  process.exit(1);
+}
+
+function createLockFile(lockFilePath) {
+  const fd = fs.openSync(lockFilePath, 'w');
+  fs.closeSync(fd);
+}
+
+function removeLockFile(lockFilePath) {
+  fs.unlinkSync(lockFilePath);
+}
+
+function lockFileExists(lockFilePath) {
+  var fileExists = false;
+  try {
+    fs.statSync(lockFilePath);
+    fileExists = true;
+  } catch (e) {
+    if (e.code !== 'ENOENT') {
+      throw e;
+    }
+  }
+  return fileExists;
```

Comment:
Does eslint get mad if you move the return into the `try`/`catch`, or do you prefer it this way? It doesn't really matter to me but I'd probably have avoided the extra var (should it be a `let`?).

```js
  try {
    fs.statSync(lockFilePath);
    return true;
  } catch (e) {
    if (e.code !== 'ENOENT') {
      throw e;
    }
    return false;
  }